climpred/smoothing.py

```diff
@@ -0,0 +1,157 @@
+import xarray as xr
+
+try:
+    import xesmf as xe
+
+    def spatial_smoothing_xesmf(ds, boxsize=(5, 5)):
+        """Apply spatial smoothing by regridding to `boxsize` grid.
+
+        Reference:
+          * Goddard, L., A. Kumar, A. Solomon, D. Smith, G. Boer, P.
+                Gonzalez, V. Kharin, et al. “A Verification Framework for
+                Interannual-to-Decadal Predictions Experiments.” Climate
+                Dynamics 40, no. 1–2 (January 1, 2013): 245–72.
+                https://doi.org/10/f4jjvf.
+
+        Args:
+            ds (xr.object): input. xr.DataArray prefered.
+            boxsize (tuple): target grid.
+                             Defaults to `(5, 5)` (see Goddard et al. 2013).
+
+        Returns:
+            ds_smoothed (xr.object): boxsize-regridded input
+
+        """
+        if isinstance(ds, xr.Dataset):
+            was_dataset = True
+            ds = ds.to_array().squeeze()
```

Comment:
I think xr.Dataset is handled now in v0.2.0
https://github.com/JiaweiZhuang/xESMF/releases/tag/v0.2.0